src/main/java/org/ibp/api/rest/study/StudyInstanceResource.java

```diff
@@ -56,7 +58,8 @@
 	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES')")
 	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/instances", method = RequestMethod.GET)
 	@ResponseBody
-	public ResponseEntity<List<StudyInstance>> listStudyEnvironments(final @PathVariable String cropname, @PathVariable final String programUUID,
+	public ResponseEntity<List<StudyInstance>> listStudyInstances(final @PathVariable String cropname,
```

Comment:
We should also add `'INFORMATION_MANAGEMENT', 'BROWSE_STUDIES'` as authorized permissions for `listStudyInstances`